### PR TITLE
Issue #7: Add connected networks & DNS server addresses

### PR DESCRIPTION
**Pull request by munirahmed007**

### CODE REVIEW

File: Menu Bar DNS/Menu Bar DNS/AppDelegate.swift [hunk 1]
```diff
@@ -6,29 +6,88 @@
...
+            for dnsAddress in wifiDNSAddresses {
+                menu?.addItem(NSMenuItem.init(title: "  \(dnsAddress)",
+                                              action: nil,
+                                              keyEquivalent: ""))
+            }
+            dnsAdded = true
+        }
+        
+        if !dnsAdded {
+            menu?.addItem(withTitle: "Hello!", action: nil, keyEquivalent: "")
```
Comment (ivanstegic) [inline Menu Bar DNS/Menu Bar DNS/AppDelegate.swift]: @munirahmed007  If there are no network connections, the message should read "Disconnected"

**Comment by ivanstegic** [comment]
@munirahmed007 it doesn't seem to be picking up the correct IPs



https://user-images.githubusercontent.com/509217/104250820-11499680-5434-11eb-9b72-273b32d9114c.mov

**Comment by munirahmed007** [comment]
that should be fixed now

File: Menu Bar DNS/Menu Bar DNS/AppDelegate.swift (hunk 1 above)
Comment (munirahmed007) [reply Menu Bar DNS/Menu Bar DNS/AppDelegate.swift]: addressed

**Comment by ivanstegic** [comment]
This all checks out, thank you.

**Pull request merged**